client/boot/index.js

```diff
@@ -323,7 +323,11 @@ function reduxStoreReady( reduxStore ) {
 			} );
 
 			if ( '/' === context.pathname && config.isEnabled( 'devdocs/redirect-loggedout-homepage' ) ) {
-				page.redirect( '/devdocs/start' );
+				if ( config.isEnabled( 'oauth' ) ) {
+					page.redirect( '/authorize' );
+				} else {
+					page.redirect( '/devdocs/start' );
```

Comment:
Hm, doesn’t it make more sense push the decision down to `devdocs/start`? If we branch here we assert that we know exactly what’s the functionality of `devdocs/start`, which we shouldn’t know :) Currently `devdocs/start` shows a link to the standard WordPress.com login, which is the place tied to the current authentication scheme.